packages/generate-pdf-template-helpers/src/helpers.ts

```diff
@@ -2,6 +2,25 @@ import { TableLayout } from 'pdfmake/interfaces';
 
 import { TableElement } from './types';
 
+/* This localizeNumber function is an exact copy of the function by the same name found at
+packages/utils/src/helpers.ts.
+This is to avoid having to import the entire utils package for only one function.
+If the utils package at some stage in the future needs to be imported for another reason,
+the function here can be removed and instead imported from packages/utils/src/helpers.ts*/
+export function localizeNumber(lvalue: number | string, locales: string | string[] = 'EN'): string {
+  if (lvalue === undefined) return '';
+
+  const options: Intl.NumberFormatOptions = {};
+  if (typeof lvalue === 'string') {
+    const decimalNumbersToDisplay = lvalue.includes('.') ? lvalue.split('.').at(-1).length : 0;
+    options.minimumFractionDigits = decimalNumbersToDisplay;
+    lvalue = Number(lvalue);
```

Comment:
Good morning! I didn't add in any extra checks because the previous implementation didn't have any checks either as you can see [here](https://github.com/s1seven/schema-tools/blob/4311c15e3847dacf5aa8ae6b066c71b1c6290962/packages/generate-pdf-template-helpers/src/helpers.ts#L75). I think it is assumed that this will only be called with a valid number. 

That being said, I had thought about adding an extra check to ensure it was a valid number and throwing a error if not. I just wasn't sure if this had been a design choice and whether it is better from a user perspective to get an error during rendering, or see `NaN` on the rendered HTML which might be easier for them to debug. Thoughts?

Reply:
Throwing an error would be a bit hard, but simply displaying the original string could have been an alternative...
Anyway let's see when the situation occurs.